### ISSUE TITLE
Photo slideshow

### ISSUE THREAD

**Issue by cpshaw99** [CRJI/theblacksea.eu#91]
THere's no slideshow function on new site: https://theblacksea.eu/index.php?idT=88&idC=88&idRec=1135&recType=photo

**Comment by nightsh** [comment]
This is right. What kind of slideshow would be ok? I'd rather avoid the old one due to some copyrighted materials in the theme used.

Is [this](https://getbootstrap.com/docs/4.0/components/carousel/#with-captions) okay @cpshaw99 ?

**Comment by germakoci** [comment]
this also seem to be discussed on the main wagtail https://github.com/wagtail/wagtail/issues/1717 

but the example of the slideshow should come from Petrut - can you please ask Petrut today @cpshaw99 ?

**Comment by cpshaw99** [comment]
I asked him today

**Comment by germakoci** [comment]
From Petrut, Photo Gallery Requirements:

The actual gallery is takes from here: https://galleria.io/. Hopefully we can use the same template.

Otherwise here are some general directions:

It should be clean, and all the buttons and captions should be outside the picture. Besides the picture
we need:

- [ ] - a caption which should be adjustable in size – but in average is one text line, maximum two
- [ ] - next/previous buttons
- [ ] -full screen button
- [ ] - navigation status (2 of 15)
- [ ] -thumbs

For uploading on the website then to the story it should be capable to do for both multiple files upload.
If you have a 20 pictures story, you should upload them once in the media library (not one by one) and
the same for adding them to the gallery itself. 

Ideally the caption should be read automatically from ITPC data. We can have them a database/archive
of images which have the captions embedded into the file – no need to have separate word doc which
usually are getting lost.

The picture should automatically resize down with fit in option, not by cropping.

**Comment by germakoci** [comment]
and linked issue #184
